scripts/setup.sh

```diff
@@ -47,7 +47,7 @@ if [[ $current_dir == */vcdat* ]]; then
     conda env remove -y -n ${CONDA_ENV}
 
     # Create a new one
-    conda create -y -n ${CONDA_ENV} -c cdat/label/nightly -c conda-forge -c cdat --file $current_dir/backend/requirements.txt
+    conda create -y -n ${CONDA_ENV} -c cdat/label/nightly -c conda-forge -c cdat
```

Comment:
I think this should read 
```
conda create -y -n ${CONDA_ENV} -c cdat/label/nightly -c conda-forge -c cdat vcdat
```

Reply:
Yeah I think you're right. I went ahead and updated that